stacks-signer/src/signerdb.rs

```diff
@@ -405,9 +415,24 @@ static SCHEMA_3: &[&str] = &[
     "INSERT INTO db_config (version) VALUES (3);",
 ];
 
+static SCHEMA_4: &[&str] = &[
```

Comment:
Check out the way I did schema changed in my PR around retrying 429'd verifications: https://github.com/stacks-network/stacks-core/pull/5453/files#diff-2142a830df37bd7d21338d998aa3c4486f0ccbab7476254db18997ef49a463c3

I don't think we should be dropping tables unless necessary, and since we're just adding an index and changing db_version, this PR can do the same approach

Reply:
Yes, I thought that was overkill, but I was just following the previous code. I'll update to your method.

Reply:
Updated in a9acfa048.